src/__tests__/__snapshots__/downshift.aria.js.snap

```diff
@@ -0,0 +1,107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`basic snapshot 1`] = `
+<div
+  aria-expanded="false"
+  aria-haspopup="listbox"
+  aria-labelledby="downshift-0-label"
+  aria-owns="downshift-0-menu"
+  role="combobox"
+>
+  <label
+    data-testid="label"
+    for="downshift-0-input"
+    id="downshift-0-label"
+  >
+    label
+  </label>
+  <input
+    aria-autocomplete="list"
+    aria-controls="downshift-0-menu"
```

Comment:
The `aria-controls="downshift-0-menu"` attribute is also throwing an error because it's referencing the unordered list of results when they're not available. If you can add/remove this attribute at the same time you add/remove the `aria-owns` attribute on the container, the error will go away.

The axe-core scanner does not report an error if Downshift is collapsed, and `aria-controls` is removed. You could tie the `aria-controls` attribute to the `aria-expanded` logic.